src/views/__tests__/TransactionSigner.test.jsx

```diff
@@ -0,0 +1,259 @@
+/**
+ * @jest-environment jsdom
+ */
+import { Networks, TransactionBuilder } from "stellar-sdk";
+import { fireEvent, waitFor, screen } from "@testing-library/react";
+import * as freighterApi from "@stellar/freighter-api";
+import albedo from "@albedo-link/intent";
+import { render } from "helpers/testHelpers";
+import TransactionSigner from "views/TransactionSigner";
+import Libify from "helpers/Libify";
+import * as transactionSignerActions from "actions/transactionSigner";
+
+jest.mock("@stellar/freighter-api", () => ({
+  isConnected: () => new Promise((resolve) => resolve(true)),
+  signTransaction: () => new Promise((resolve) => resolve(SIGNED_TRANSACTION)),
+}));
+
+jest.mock("@albedo-link/intent", () => ({
+  tx: () =>
+    new Promise((resolve) =>
+      resolve({ signed_envelope_xdr: SIGNED_TRANSACTION }),
+    ),
+}));
+
+const TRANSACTION_XDR =
+  "AAAAAgAAAADYMr0vKfmdLXQTWwns9YavTU2ZCDzl7+L/dWEqn05dqQAAAGQAABjxAAAACwAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAABQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const FEE_BUMP_XDR =
+  "AAAABQAAAACafGsfMQylseeF9Du5owr05nhq/IDVl47HGz6PHGwWfwAAAAAAAADIAAAAAgAAAADYMr0vKfmdLXQTWwns9YavTU2ZCDzl7+L/dWEqn05dqQAAAGQAABjxAAAACwAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAABQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```

Comment:
Using a couple of test transactions so we can confidently say that static inputs always result in the same output